db/migrate/20210831100856_remove_dangling_accounts.rb

```diff
@@ -1,6 +1,6 @@
 class RemoveDanglingAccounts < ActiveRecord::Migration[5.2]
   def up
-    DanglingAccountRemover.run!
+    # DanglingAccountRemover.run!
```

Comment:
Wouldn't it make sense to remove this entirely ? I don't know if it's possible (if that will break other stuff). If it doesn't I'd remove the file entirely

Reply:
No, all migrations are kept in the schema DB and that can cause issues.